content/about/index.md

```diff
@@ -1,6 +1,5 @@
 +++
 title = "About"
-template = "page.html"
```

Comment:
page.html is the default template, no need to specify it.

For about and subscribe, if you don't want to put content next to it, there is no need to put them in a directory. This file is equivalent to `content/about.md` for example.

Reply:
Got it.  Since I don't have any content so far I'll rename the md files.